.github/workflows/pyflakes.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - development
+      - 'release-candidate-*'
+
+jobs:
+  pyflakes:
+    name: runner / pyflakes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: pyflakes
+        uses: reviewdog/action-pyflakes@v1
```

Comment:
What's the reason to run the linter via this action? After looking at https://github.com/reviewdog/action-pyflakes it seems not very popular. Also it shouldn't be that hard to come up with a valid command to lint the necesarry directories 🤔

Reply:
running the command is easy. The reason to use a GitHub Action for it is to integrate the _results_ with how GitHub presents the PR. It inlines them with the review.